application/views/dashboard/index.php

```diff
@@ -108,7 +108,7 @@ function getDistance($distance) {
 
 	<?php if (version_compare(PHP_VERSION, '7.4.0') <= 0) { ?>
 		<div class="alert alert-danger alert-dismissible fade show" role="alert">
-			<?= __("You need to upgrade your PHP version. Minimum version is 7.4. Your version is") . ' ' . PHP_VERSION . '.';?>
+			<?= __("You need to upgrade your PHP version. Minimum version is 8.0. Your version is") . ' ' . PHP_VERSION . '.';?>
```

Comment:
Shall we make this string dynamic so we don't have to retranslate it in the future over and over again?


```suggestion
			<?= sprintf(__("You need to upgrade your PHP version. Minimum version is %s. Your version is: %s."), "8.0", PHP_VERSION); ?>
```